sources/HUBViewModelDiff.m

```diff
@@ -145,15 +164,237 @@ + (instancetype)diffFromViewModel:(id<HUBViewModel>)fromViewModel
                                              reloads:[reloads copy]];
 }
 
-- (NSString *)debugDescription
+typedef struct {
+    NSInteger x;
+    NSInteger y;
+} HUBDiffPoint;
+
+static inline HUBDiffPoint HUBDiffPointMake(NSInteger x, NSInteger y) {
+    return (HUBDiffPoint){ .x = x, .y = y};
```

Comment:
🎨 Style is a bit inconsitent here, you have a leading space but not a trailing one 😄